vendor/github.com/docker/go-connections/tlsconfig/config.go

```diff
@@ -13,6 +13,9 @@ import (
 	"os"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/notary"
```

Comment:
Thank you for the review. I will update the PR to remove the dependency on notary and include the TLS keys handling logic in go-connections instead.